src/pages/homeCollection/kitAssembly.js

```diff
@@ -204,7 +204,7 @@ const renderSidePane = () => {
         Return Kit ID = ${ kitObject[conceptIds.returnKitId] } |
         Cup Id = ${ kitObject[conceptIds.collectionCupId] } |
         Card Id = ${ kitObject[conceptIds.collectionCardId] }
-        <button type="button" class="btn btn-outline-primary detailedRow" data-kitObject=${JSON.stringify(kitObject)} id="editAssembledKits">Edit</button>
+        <button type="button" class="btn btn-outline-primary detailedRow" data-kitObject=${encodeURIComponent(JSON.stringify(kitObject))} id="editAssembledKits">Edit</button>
```

Comment:
I feel storing  data to and retrieving data from webpage are not ideal. Maybe these are something to be polished later.